src/components/subscriptions/SubscriptionData.jsx

```diff
@@ -81,6 +83,17 @@ export default function SubscriptionData({ children }) {
       </SubscriptionContext.Provider>
     );
   }
+  if (!isLoading) {
+    return (
+      <StatusAlert
+        className="mt-3"
+        alertType="danger"
+        message={`Your organization does not have any active subscriptions to manage.
+        If you believe you are seeing this message in error,
+        please reach out to the edX Customer Success team at customersuccess@edx.org.`}
+      />
+    );
```

Comment:
@irfanuddinahmad  As per my understanding, we need to make this decision on `SubscriptionManagementPage` and either display the  usual UI or error message.